tests/test_tags.py

```diff
@@ -271,18 +285,31 @@ def test_mac_platforms(self):
             "macosx_10_5_intel",
             "macosx_10_5_fat64",
             "macosx_10_5_fat32",
+            "macosx_10_5_universal2",
             "macosx_10_5_universal",
             "macosx_10_4_x86_64",
             "macosx_10_4_intel",
             "macosx_10_4_fat64",
             "macosx_10_4_fat32",
+            "macosx_10_4_universal2",
             "macosx_10_4_universal",
         ]
 
-        assert len(list(tags.mac_platforms((10, 17), "x86_64"))) == 14 * 5
+        assert len(list(tags.mac_platforms((10, 17), "x86_64"))) == 14 * 6
 
         assert not list(tags.mac_platforms((10, 0), "x86_64"))
 
+    def test_macos_11(self):
+        platforms = list(tags.mac_platforms((11, 0), "x86_64"))
+        assert "macosx_10_15_x86_64" in platforms
+        assert "macosx_10_4_x86_64" in platforms
+        assert "macosx_10_3_x86_64" not in platforms
+
+        platforms = list(tags.mac_platforms((11, 0), "arm64"))
+        assert "macosx_10_15_x86_64" not in platforms
+        assert "macosx_10_4_x86_64" not in platforms
+        assert "macosx_10_3_x86_64" not in platforms
```

Comment:
Can we add positive assertions as well for what _is_ the expected platform tags?

Reply:
Good thing you asked!   I realized when adding some that this was right for 11.0, but not so right for 11.3 or 12.0.